services/ext/api.go

```diff
@@ -978,14 +978,8 @@ func (api *PublicAPI) RegisteredForPushNotifications() (bool, error) {
 }
 
 // Emoji
-
-func (api *PublicAPI) SendEmojiReaction(ctx context.Context, chatID, messageID string, emojiID protobuf.EmojiReaction_Type) (*protocol.MessengerResponse, error) {
-	return api.service.messenger.SendEmojiReaction(ctx, chatID, messageID, emojiID, "")
-}
-
-// TODO remove SendEmojiReaction and remove the V2 when the client supports sending custom emojis
-func (api *PublicAPI) SendEmojiReactionV2(ctx context.Context, chatID, messageID string, emoji string) (*protocol.MessengerResponse, error) {
```

Comment:
Shouldn't only `SendEmojiReactionV2` remain? Or you will adapt the desktop side as well?